src/sentry/snuba/metrics/fields/base.py

```diff
@@ -1397,6 +1405,25 @@ def generate_where_statements(
                 crash_user_rate_value, alias=alias
             ),
         ),
+        SingularEntityDerivedMetric(
+            metric_mri=SessionMRI.CRASH_FREE_USER.value,
+            metrics=[
+                SessionMRI.ALL_USER.value,
+                SessionMRI.CRASHED_USER.value,
+            ],
+            unit="users",
+            snql=lambda all_user_count, crashed_user_count, project_ids, org_id, metric_ids, alias=None: subtraction(
+                all_user_count, crashed_user_count, alias=alias
+            ),
+        ),
+        SingularEntityDerivedMetric(
+            metric_mri=SessionMRI.CRASH_FREE_USER.value,
+            metrics=[SessionMRI.CRASH_USER_RATE.value],
+            unit="percentage",
+            snql=lambda crash_user_rate_value, project_ids, org_id, metric_ids, alias=None: complement(
+                crash_user_rate_value, alias=alias
+            ),
+        ),
```

Comment:
I assume this has been duplicated by mistake from `CRASH_FREE_USER_RATE`.

Reply:
Need to check what's going on here, thanks